File: Week-1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
  int height;

  do {
    height = get_int("How high? (1-8)\n");
  } while (height < 1 || height > 8);

  for (int i = 1; i <= height; i++) {
    for (int j = height; j > i; j--) {
      printf(" "); // Spaces
    }
    for (int k = 1; k <= i; k++) {
      printf("#"); // Hashtags
    }
    printf("\n");
  }
}

File: pset6/mario.py
from cs50 import get_int 

def main():
    while True:
        height = get_int('Height: ')
        if height >= 1 and height <= 8:
            break
        else:
            print("Please enter a height between 1 and 8.")
            continue # Ask again in case of invalid input

    for i in range(1, height + 1):
        # Calculate the required number of spaces and # for each line
        spaces = height - i
        hashes = i
        print(' ' * spaces + '#' * hashes)

if __name__ == "__main__": 
    main()